tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/amqp/AmqpClientTestSupport.java

```diff
@@ -155,6 +155,7 @@ protected ActiveMQServer createServer(int port) throws Exception {
       server.getConfiguration().setPagingDirectory(server.getConfiguration().getPagingDirectory() + port);
       server.getConfiguration().setJMXManagementEnabled(true);
       server.getConfiguration().setMessageExpiryScanPeriod(5000);
+      server.getConfiguration().setUseLegacyAMQPQueueNaming(false);
```

Comment:
I don't think the tests should override the default like this, it means none of the tests would be testing the default behaviour of the broker. Explicit tests that verify non-default behaviours would be more appropriate.